Header header
string name
IntersectionV2X intersection
int32 revision
string status
int32 MinuteOfYear
int32 timeStamp
int32 lengthOfStates

SPATStates[] states
